draft-packages/badge/docs/Badge.stories.tsx

```diff
@@ -56,141 +56,59 @@ const BadgeStoryWrapper: React.FunctionComponent<{
   )
 }
 
-export const DefaultStory = args => <Badge {...args}>3</Badge>
-
-DefaultStory.storyName = "Default (with args)"
-DefaultStory.args = {
-  variant: "default",
-  reversed: false,
-  size: "small",
-}
+const BadgeStoryWrapper: React.FunctionComponent<{
+  children
+}> = ({ children }) => <div style={{ padding: "20px" }}>{children}</div>
 
-export const AnimatedBadge = args => (
-  <>
-    <BadgeStoryWrapper>
-      {(badgeCount, useAnimation) =>
-        useAnimation ? (
-          <BadgeAnimated {...args}>{badgeCount}</BadgeAnimated>
-        ) : (
-          <Badge {...args}>{badgeCount}</Badge>
-        )
-      }
-    </BadgeStoryWrapper>
-  </>
+export const DefaultStory = args => (
+  <BadgeAnimationStoryWrapper>
+    {(badgeCount, useAnimation) =>
+      useAnimation ? (
+        <BadgeAnimated {...args}>{badgeCount}</BadgeAnimated>
+      ) : (
+        <Badge {...args}>{badgeCount}</Badge>
+      )
+    }
+  </BadgeAnimationStoryWrapper>
 )
+DefaultStory.storyName = "Default (Kaizen Demo)"
 
-AnimatedBadge.description = "skdjbnsdkjfnsdf lsndfksdjlfn skdjfn"
-
-export const LightBadges = () => (
-  <>
-    <Heading style={{ marginBottom: "2rem" }} variant="heading-2" tag="h1">
-      Light Badges
-    </Heading>
-    <Heading variant="heading-3" tag="h2">
-      Default
-    </Heading>
-    <div className={styles.badgeSection}>
-      <div className={styles.badgeSize}>
-        <Heading variant="heading-5" tag="h3">
-          Small
-        </Heading>
-        <Badge variant="default">3</Badge>
-      </div>
-      <div className={styles.badgeSize}>
-        <Heading variant="heading-5" tag="h3">
-          Large
-        </Heading>
-        <Badge size="large" variant="default">
+const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
+  isReversed,
+}) => (
+  <StoryWrapper isReversed={isReversed}>
+    <StoryWrapper.RowHeader headings={["Default", "Active", "Dot"]} />
+    <StoryWrapper.Row rowTitle="Small">
+      <BadgeStoryWrapper>
```

Comment:
I think it's fine to just not add the extra padding around the small badges. It actually throws off the alignment a little the more I look at it (since it's neither left nor centre aligned) 🤔 
Also, and this may only apply to me, but I would question whether the excess white space would be baked into the component when I see it, which is a check I shouldn't need to do.

I think we should only need to decorate our stories if they are not clear without them :)